index.html

```diff
@@ -2314,16 +2294,20 @@ <h3>The <dfn>NFCReaderOptions</dfn> dictionary</h3>
                 <ol>
                   <li>
                     If <var>MIME type</var>.parameters[<code>"charset"</code>]
-                    <a>exists</a>, set <var>charset</var> to the value.
+                    <a>exists</a>, set <var>charset</var> to the value, <a>lowercased</a>.
```

Comment:
ASCII lowercase?

Reply:
`as ASCII lowercase`?  or `to the result of running ASCII lowercase on value?`

Reply:
"ASCII lowercased" would be fine, was mostly wondering what lowercased meant.

Reply:
fixed